src/cssm.pyx

```diff
@@ -3322,6 +3322,218 @@ def ddm_flexbound_mic2_ornstein_multinoise(np.ndarray[float, ndim = 1] vh,
         raise ValueError('return_option must be either "full" or "minimal"')
 # ----------------------------------------------------------------------------------------------------
 
+
+# Simulate (rt, choice) tuples from: Vanilla LBA Model without ndt -----------------------------
+def lba_vanilla_wo_ndt(np.ndarray[float, ndim = 2] v, 
```

Comment:
ok just one last comment (should have mentioned this earlier that slipped through). 

At best, keep the cython level implementation so that it includes non-decision time, but set the non-decision time to 0, at the level of `simulator()` function. The idea is to keep the cython stuff a bit general, so we can reuse it for multiple derivative models. There are a bunch of examples for that with other models which may have a `no_bias` version in model config, but it simply calls the same cython level simulator as the fully flexible version.